### DISCUSSION TITLE
Medusa Torrent snatch directory

### DISCUSSION THREAD

**Discussion by mtclemente** [pymedusa/Medusa#9086] [Q&A]
I keep getting an error message in the logs:
`2021-01-22 10:21:19 WARNING  SNATCHQUEUE-SNATCH-361315 :: [e6025de] Torrent file content is empty: Batwoman.S01E12.1080p.BluRay.x264-SHORTBREHD`

I want to clear/delete that torrent out so that it stops sending it to my downloader (transmission) since it never downloads because it's an empty torrent.  How do I delete this file or what directory is it stored in so I can remove it and it stops sending to my downloader?  I have CLI access to Medusa...just don't know where it stores the .torrent files before sending to transmission.

**Comment by medariox** [comment]
You can set any directory if you use Blackhole method. If you leave it empty the torrent files will be stored in Medusas main directory.

**Reply by mtclemente** [reply]
in /app/medusa?  What folder in there are they stored in?  I did a 'find' and can't find any .torrent files.  Not familiar with the 'Blackhole' method.

**Reply by medariox** [reply]
If you aren't using Blackhole, but direct client connection, torrent files aren't saved anywhere. If they fail (like above) they are just deleted directly.

**Reply by mtclemente** [reply]
If they are deleted directly, why does it keep trying to snatch/download this old torrent?  That episode is marked as downloaded and of an acceptable quality...but Medusa still continues to search for it and tries to download it.  How do I clear that SnatchQueue so that it stops trying to download this episode?

**Reply by medariox** [reply]
Probably because that torrent has a preferred quality over what you already have. Check your quality settings.

**Reply by mtclemente** [reply]
Nope...Quality is 720/1080...that's what I meant by an acceptable quality...the file is marked as a 720p file.  It should not be searching for it...  Yet it still keeps trying to find just this one specific episode...

![image](https://user-images.githubusercontent.com/9056865/105534217-28517980-5cbb-11eb-89a2-f44fb4717d11.png)

![image](https://user-images.githubusercontent.com/9056865/105534254-3606ff00-5cbb-11eb-8d26-013213a86093.png)

**Reply by mtclemente** [reply]
![image](https://user-images.githubusercontent.com/9056865/105534368-60f15300-5cbb-11eb-833e-8503409e26fc.png)

**Comment by medariox** [comment]
Yep, set like that it will always download the highest quality. Read https://github.com/pymedusa/Medusa/wiki/Quality-Settings#custom-qualities

**Reply by mtclemente** [reply]
No, it will stop when it finds the first quality in the allowed list...ie 720p.  It says so in your link...and right underneath it:

![image](https://user-images.githubusercontent.com/9056865/105536611-9e0b1480-5cbe-11eb-90e6-17b9d53ca2b4.png)

**Reply by medariox** [reply]
Open an issue with related debug logs of the snatch.
